apis/duck/patch.go

```diff
@@ -20,7 +20,7 @@ import (
 	"encoding/json"
 
 	jsonmergepatch "github.com/evanphx/json-patch"
-	"github.com/mattbaird/jsonpatch"
+	jsonpatch "gomodules.xyz/jsonpatch/v2"
```

Comment:
For posterity, this is a fork of what we were using: https://github.com/gomodules/jsonpatch

Reply:
Wonder what made you drop it?